package.json

```
@@ -4,6 +4,10 @@
   "description": "100% typesafe dependency injection framework for TypeScript projects",
   "license": "MIT",
   "author": "Snap Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Snapchat/ts-inject.git"
```

according to [this](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository) it seems this value has to be
```suggestion
    "url": "git+https://github.com/Snapchat/ts-inject.git"
```

---

Done!